### DISCUSSION TITLE
Tutorial Section

### DISCUSSION THREAD

**Discussion by Dr-Electron** [iotaledger/iota-wiki#457] [Ideas]
The wiki team is working on creating a tutorial section together with the IOTA foundation
Let's discuss here how it should work and look like.

You can take this first [PoC](https://dr-electron.github.io/iota-wiki/showcase) as a point of reference but keep in mind it's just a very early draft

Now my current idea is to put the tutorials itself into their own separate repos. That way the wiki can pull the tutorials from all the repos like it currently does with the build section.
The only thing needed in the repo is a docs folder filled with the content for the wiki. We could enforce styling an repo structure with templates and styling guides.
The templates would make it easy for content creators to getting started fast. 
Template repo could contain the docs folder, a basic GitPod config, wiki-cli for easy deployment testing if needed, basic framework related setups (SCs, Identity...) and so on.
The content creator would then just need to provide a link to his repo in the wikis config and add some tags like language, difficulty and so on.

We will start integrating the stuff discussed here into a dev build of the wiki while we speak.
Some of my open questions are:
- [ ] How do we make sure the content we pull is save?
- [ ] What with simple tutorials that may not need a whole repo?
- [x] What should the section contain? Also stuff for people that aren't devs? What would we call it then? Academy?
    - Will be Tutorials for now

**Comment by Dr-Electron** [comment]
One of the ideas to solve question 1 was to fork the tutorials into the iota-community org. That way we have are able to control the tutorial content.

**Reply by DiePutze5853** [reply]
I really like the idea of the template. 
I think the tutorial content could be managed/reviewed either via the x-teams or the content-creator DAO and the IOTA Alpha Developer DAO.
I think the tutorial section should contain both articles for end-user and developer

**Reply by rafaeldjpbrochado** [reply]
Yeah maybe there's a "community tutorials" repo under the iota-community org and people can just submit PRs on there which then have to be approved & merged before the wiki displays them?

**Reply by Dr-Electron** [reply]
The problem is mostly if we approve and the maintainer upates adain without us approving again something bad could happen. With this solution we are able to see and control updates which i think is nice. But it will create a lot of repos if people really start creating tutorials. Because we basically need 1repo = 1 tutorial to add features like Gitpod

**Comment by rafaeldjpbrochado** [comment]
Regarding question 3: the only thing I'd suggest at the moment would be clearly separating community video tutorials/streams from community written tutorials. Also, perhaps there can be a subsection within "community videos" for non-tutorials as well, like for example IOTA Weekly.

**Reply by Dr-Electron** [reply]
Yeah. We need to come up with some useful tags. Otherwise if we have to much random stuff in there it won't help people looking for what they need

**Comment by anistark** [comment]
Let's get [this](https://dr-electron.github.io/iota-wiki/showcase) going?

**Reply by Dr-Electron** [reply]
What do you mean? It's finished. Don't you like the tutorials? :trollface: 
Btw we are already working on integrating some tutorials to give people and ourselves a feeling how it would really work

**Reply by anistark** [reply]
Open PR or push it? 

Then we can start adding to it.

**Reply by Dr-Electron** [reply]
You can just make PRs against [my repo](https://github.com/dr-electron/iota-wiki/tree/feat/tuto-section). Until then it's convenient to keep it in it's own repo for publishing it.

**Reply by anistark** [reply]
Yeah, I meant let's merge it and get this going live. It's the natural way to go. So, if we're waiting for a vote on it, here's my __+1__ 👍🏼  
😄

**Reply by anistark** [reply]
Btw, right now, we got this section alone: https://wiki.iota.org/build/tutorials/youtube

So, all these can go here immediately.

**Reply by Dr-Electron** [reply]
Got it :)
I just want to resolve the open questions first ;)

Also good point with the videos. I added them. Two of them as playlists. Not sure if that makes sense 🤔

**Comment by Dr-Electron** [comment]
We have a straight forward way to solve this in the [feat/tuto-section](https://github.com/iota-community/iota-wiki/tree/feat/tuto-section) branch. So we can close the discussion
